src/screens/RestoreSheet.js

```diff
@@ -80,49 +79,41 @@ export function RestoreSheet() {
     }
   }, [setParams]);
 
-  const onManualRestore = useCallback(() => {
-    analytics.track('Tapped "Restore with a secret phrase or private key"');
-    InteractionManager.runAfterInteractions(goBack);
-    InteractionManager.runAfterInteractions(() => {
-      setTimeout(() => navigate(Routes.IMPORT_SEED_PHRASE_FLOW), 50);
-    });
-  }, [goBack, navigate]);
-
-  const onWatchAddress = useCallback(() => {
-    analytics.track('Tapped "Watch an Ethereum Address"');
-    InteractionManager.runAfterInteractions(goBack);
-    InteractionManager.runAfterInteractions(() => {
-      setTimeout(() => navigate(Routes.IMPORT_SEED_PHRASE_FLOW), 50);
-    });
-  }, [goBack, navigate]);
+  const [sheetHeight, setSheetHeight] = useState(0);
 
-  const wrapperHeight =
-    deviceHeight +
-    longFormHeight +
-    (IS_ANDROID ? getSoftMenuBarHeight() / 2 : 0);
+  useEffect(() => setParams({ longFormHeight: sheetHeight }), [
+    setParams,
+    sheetHeight,
+  ]);
 
   return (
-    <Column height={wrapperHeight}>
```

Comment:
How come no height here now?

Reply:
i just set the height of the SlackSheet itself, not sure why this was necessary

Reply:
Ben is smart, I like how this works